### ISSUE TITLE
Provide example getting from the linear results to the DN numbers

### ISSUE THREAD

**Issue by joroeder** [oemof/DHNx#104]
So, I now think to know the problem of [https://github.com/oemof/DHNx/issues/66#issuecomment-1066173647](https://github.com/oemof/DHNx/issues/66#issuecomment-1066173647). @LuixLore You have the same heatpipetype, because this heatpipe type is the linearization of multiple DN numbers. In your postprocessing, you should map back the results of the capacity, to your assumptions of which DN number can transport which capacity.

We should/could add an exmaple for that.

**Comment by gitfeldhase** [comment]
I agree, an example how to recalculate the DN number for each pipe leading to the required output for a following simulation would be a great extension to the existing examples.

**Comment by gitfeldhase** [comment]
Maybe it is possible to offer a general approach with some common values of existing grids which pipes diameter are used for a certain capacity and temperature level / delta temperature.
